packages/styling/package.json

```diff
@@ -9,7 +9,9 @@
   "license": "MIT",
   "main": "lib-commonjs/index.js",
   "module": "lib/index.js",
-  "sideEffects": true,
+  "sideEffects": [
```

Comment:
@kenotron do you know more about the background for having sideEffects: true here?

Reply:
I do; the concern was mainly fontFace registration getting dropped. Working with Xu on this.